python/server/qgsserverprojectutils.sip

```diff
@@ -123,6 +123,30 @@ namespace QgsServerProjectUtils
       */
     QString wfsServiceUrl( const QgsProject& project );
 
+    /** Returns the Layer ids list defined in a QGIS project as published in WFS.
+      * @param project the QGIS project
+      * @return the Layer ids list.
+      */
+    QStringList wfsLayerIds( const QgsProject& project );
```

Comment:
Probably yes, but has to be done in an other PR after the #4313 will have been merged